### DISCUSSION TITLE
Moment-timezone vulnerability

### DISCUSSION THREAD

**Discussion by SuperBlahman** [sequelize/sequelize#15684] [Help & Questions] [answered]
Any chance of updating the version of moment-timezone this project uses? 0.5.34 has a major listed vulnerability that is patched in 0.5.35 or later. More details here: https://github.com/moment/moment-timezone/commit/7915ac567ab19700e44ad6b5d8ef0b85e48a9e75

**Comment by WikiRik** [comment, marked as answer]
`moment-timzone` is not pinned to a specific version so re-installing the dependencies should fix this. There might be nicer tools, but removing the lock file and node_modules folder and installing the dependencies should work just fine

**Reply by SuperBlahman** [reply]
True, but bumping up the minimum version sequelize uses would save some headache on any end user's part, especially if they weren't aware of the vulnerability. Better to have the patched version ootb, isn't it?

**Reply by WikiRik** [reply]
True. Feel free to open a PR so we can include it in the next release

**Reply by SuperBlahman** [reply]
Sure, find it here if interested: https://github.com/sequelize/sequelize/pull/15685
